docs/cristin.ttl

```diff
@@ -208,42 +226,50 @@
 
 
 :brukernavn_opprettet
-    a            rdf:Property ;
+    a            :cristinfield ;
     rdfs:comment """
-       eierkode_opprettet and brukernavn are the primary key for idenitfying which user created the entry. As a general rule a physical person
+        When the domain is :Result, eierkode_opprettet and brukernavn are the primary key for idenitfying which user created the entry. As a general rule a physical person
         can have many usernames (brukernavn) but only one eierkode. This holds even if a physical person moves from institution to institution
         and/or changes roles. Duplicates that violate this assumption are possible to occur but they are not expected to be the rule.
+
+        When it is :VITENSKAPELIGARBEID_LOKAL_Class it is the user who created the NVI entry."
+        This entails that the semantics of this term are unclear, possibly even polysemous.
```

Comment:
Well, as discussed, the semantics include the containing object, not the JSON fieldname alone. "brukernavn_opprettet" is whoever was logged in to create that object, be it the result itself or some of its attached objects.